service-registrar/src/main/java/com/msm/sr/service/status/notification/manager/ServiceStatusNotifierManager.java

```diff
@@ -8,10 +8,13 @@
  */
 public class ServiceStatusNotifierManager {
 
-    private static final ServiceStatusNotifier SERVICE_STATUS_NOTIFIER = new ServiceStatusNotifier();
+    private static final ServiceStatusNotifier serviceStatusNotifier = new ServiceStatusNotifier();
+
+    private ServiceStatusNotifierManager() {
```

Comment:
The private constructor is to avoid creating instances. The manager class should be singleton